eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -449,6 +455,13 @@ steps:
     artifactName: "localizeInputs"
   condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
 
+- task: PublishPipelineArtifact@1
+  displayName: "Upload legacy localizeInputs artifact"
+  inputs:
+    targetPath: "$(Build.Repository.LocalPath)\\artifacts\\**\\*.lcg;!$(Build.Repository.LocalPath)\\artifacts\\localize\\**\\*"
+    artifactName: "legacy localizeInputs"
+  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'))"
```

Comment:
Official only?

Reply:
I intended to leave out official for testing, and once validated, put it in before merging. I'll add a comment in my next push to make it obvious, and hopefully remember to fix before merging.